Toolkit/ArcGISToolkit/MeasureToolbar.swift

```diff
@@ -297,30 +311,14 @@ public class MeasureToolbar: UIView, AGSGeoViewTouchDelegate {
         
         // notification
         
-        NotificationCenter.default.addObserver(forName: NSNotification.Name.AGSSketchEditorGeometryDidChange, object: lineSketchEditor, queue: nil, using: sketchDidChange)
```

Comment:
This (and the subsequent line) were also preventing the measure toolbar from being deallocated.